package.json

```diff
@@ -65,11 +68,12 @@
     "@emotion/react": "^11.7.1",
     "@emotion/styled": "^11.6.0",
     "@eps1lon/enzyme-adapter-react-17": "^0.1.0",
-    "@material-ui/monorepo": "https://github.com/mui-org/material-ui.git#master",
```

Comment:
@flaviendelangle FYI, I think it makes more sense to update `package.json` rather than `yarn.lock`.

Reply:
Okay, I will revert the change.